backoffice/views.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.views import LoginView as BaseLoginView
+from django.shortcuts import resolve_url
+
+
+class LoginView(BaseLoginView):
+    """
+    A custom login view to the admin section.
```

Comment:
peut etre expliquer le but de la manoeuvre ici : "We don't use the default admin login view because it uses "?next=" redirections which are not safe."

Aussi peut etre l'appeler AdminLoginView, pour pas se melanger les pedales avec la LoginView de magicauth

Reply:
ok